authorization.md

```diff
@@ -171,16 +226,37 @@ The access token response is a JSON object, with the following properties:
   </tbody>
 </table>
 
-## Server Obligations ##
+## Server Obligations for Signature Verification
 
-<span class="label label-warning">TODO</span>
+Servers SHALL follow all requirements defined in [Section 3 of RFC7523](https://tools.ietf.org/html/rfc7523#section-3).
 
-Servers SHALL
+In addition, we require that servers SHALL:
 * validate the signature on the JWT
-* check that the JWT `exp` is valid
-* check that the JWT `aud` matches the server's OAuth token URL (the URL to which the token was `POST`ed)
+* check that the JWT `exp` claim is valid
+* check that the JWT `aud` claim matches the server's OAuth token URL (the URL to which the token was `POST`ed)
 * check that this is not a `jti` value previously encountered for the given `sub` within the maximum allowed authentication JWT lifetime (5 minutes). This check prevents replay attacks.
```

Comment:
the jti check should be for the given `iss` (not `sub`) even if iss and sub happen to be the same.

Reply:
Agreed - will fix. Thanks!